### PR TITLE
[rastertools/tilesxyzdirectory] Document new parameters

### PR DESCRIPTION
**Pull request by pathmapper**
Document three new optional advanced parameters (QGIS 3.30), ref https://github.com/qgis/QGIS/pull/51522.

Also: Remove duplicated output entries.

### CODE REVIEW

File: docs/user_manual/processing_algs/qgis/rastertools.rst [hunk 1]
```diff
@@ -305,47 +311,85 @@ Parameters
 
        Default: False
      - 
-   * - **Output directory**
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/rastertools.rst:308]: These shouldn't be removed/moved

**Review by DelazJ — COMMENTED** [review]
> Also: Remove duplicated output entries.

These are not duplicate: in one case it is an input parameter the user needs to fill (generally a path to a folder/file), and in the other case, it is the result/output they get i.e., the new layer, file, folder, with -- and this is where we can do more/better --- the details of what they really get (what new fields, layer type, what in the folder...)

File: docs/user_manual/processing_algs/qgis/rastertools.rst [hunk 2]
```diff
@@ -305,47 +311,85 @@ Parameters
...
+       HTML links are possible.
+   * - **Include OpenStreetMap basemap in Leaflet HTML output**
+
+       Optional
+     - ``HTML_OSM``
+     - [boolean]
+
+       Default: False
+     - An OpenStreetMap basemap layer (source: https://tile.openstreetmap.org)
+       is included in the Leaflet HTML outut file. Proper map attribution is
```
[the remark this answers is not in the document]
Comment (pathmapper) [reply docs/user_manual/processing_algs/qgis/rastertools.rst]: Thanks, addressed per https://github.com/qgis/QGIS-Documentation/pull/7988/commits/3e932b3c5742595434d112d0bb3ec761fcf65aa7

File: docs/user_manual/processing_algs/qgis/rastertools.rst (hunk 1 above)
Comment (pathmapper) [reply docs/user_manual/processing_algs/qgis/rastertools.rst:308]: Thanks for clarification 👍  put them back in https://github.com/qgis/QGIS-Documentation/pull/7988/commits/3e932b3c5742595434d112d0bb3ec761fcf65aa7.

**Pull request merged**